board.go

```diff
@@ -65,14 +65,14 @@ type SprintsList struct {
 
 // Sprint represents a sprint on Jira agile board
 type Sprint struct {
-	ID            int        `json:"id" structs:"id"`
-	Name          string     `json:"name" structs:"name"`
-	CompleteDate  *time.Time `json:"completeDate" structs:"completeDate"`
-	EndDate       *time.Time `json:"endDate" structs:"endDate"`
-	StartDate     *time.Time `json:"startDate" structs:"startDate"`
-	OriginBoardID int        `json:"originBoardId" structs:"originBoardId"`
-	Self          string     `json:"self" structs:"self"`
-	State         string     `json:"state" structs:"state"`
+	ID            int        `json:"id,omitempty" structs:"id,omitempty"`
+	Name          string     `json:"name,omitempty" structs:"name,omitempty"`
+	CompleteDate  *time.Time `json:"completeDate,omitempty" structs:"completeDate,omitempty"`
+	EndDate       *time.Time `json:"endDate,omitempty" structs:"endDate,omitempty"`
+	StartDate     *time.Time `json:"startDate,omitempty" structs:"startDate,omitempty"`
+	OriginBoardID int        `json:"originBoardId,omitempty" structs:"originBoardId,omitempty"`
+	Self          string     `json:"self,omitempty" structs:"self,omitempty"`
+	State         string     `json:"state,omitempty" structs:"state,omitempty"`
 }
```

Comment:
`Sprint` struct has been used only in unmarshaling JSON responses from /boards API call.
So adding `omitempty` will not break any of current behaviours.